### PR TITLE
[backport] My-account-page-Recently-Ordered-check-box-misaligned-on-tab-portrait…

### PR DESCRIPTION
**Pull request by amol2jcommerce**
### Description (*)

#20143 : My account page "Recently Ordered" check box misaligned on tab portrait view
### Fixed Issues (if relevant)

1. #20155: My account page "Recently Ordered" check box misaligned on tab portrait view

### Manual testing scenarios (*)

1. placed order with login customer on my account page in sidebar check resend order product.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @amol2jcommerce. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.2-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Comment by dmytro-ch** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, here is your [new Magento instance](https://pr-21171.instances.magento-community.engineering).
Admin access: [https://pr-21171.instances.magento-community.engineering/admin](https://pr-21171.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

File: app/design/frontend/Magento/luma/Magento_Sales/web/css/source/_module.less [hunk 1]
```diff
@@ -343,16 +343,24 @@
         }
 
         .product-item-name {
-            display: inline-block;
+            float: left;
+            width: calc(100% - 20px);
```
Comment (dmytro-ch) [inline app/design/frontend/Magento/luma/Magento_Sales/web/css/source/_module.less]: The `calc` operator is used not properly here.
It won't give the expected result.
Currently, it results with value "80%".

File: app/design/frontend/Magento/luma/Magento_Sales/web/css/source/_module.less [hunk 2]
```diff
@@ -343,16 +343,24 @@
...
         }
 
         .product-item {
+            &:after {
```
Comment (dmytro-ch) [inline app/design/frontend/Magento/luma/Magento_Sales/web/css/source/_module.less]: I guess we can use `.lib-clearer()` mixin instead.

**Review by dmytro-ch — CHANGES_REQUESTED** [review]
Hi @amol2jcommerce,
thank you for your contribution.
Could you please fix the issues mentioned in the review?

File: app/design/frontend/Magento/luma/Magento_Sales/web/css/source/_module.less (hunk 1 above)
Comment (amol2jcommerce) [reply app/design/frontend/Magento/luma/Magento_Sales/web/css/source/_module.less]: thanks

**Comment by amol2jcommerce** [comment]
Hi @dmytro-ch branch updated. please review.

**Comment by dmytro-ch** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, here is your [new Magento instance](https://pr-21171.instances.magento-community.engineering).
Admin access: [https://pr-21171.instances.magento-community.engineering/admin](https://pr-21171.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-4243](https://jira.corp.magento.com/browse/ENGCOM-4243) has been created to process this Pull Request

**Comment by sivaschenko** [comment]
Hi @amol2jcommerce, we are not able to reproduce this issue on 2.2-develop, can you please double check if the fix is needed for this version.
image

**Comment by amol2jcommerce** [comment]
Hi @sivaschenko, I recheck & this issue on 2.2-develop not reproduce.

Thanks

**Pull request closed without merging**